According to modern desktop entry specifications, keywords
must be separated by and be terminated with a semicolon.

Additionally, the change is required for modern freedesktop
consumers to make use of the keywords.
--- keurocalc/keurocalc.desktop
+++ keurocalc/keurocalc.desktop
@@ -103,50 +103,50 @@
 Comment[x-test]=xxA currency converter and calculatorxx
 Comment[zh_CN]=一个汇率转换器和计算器
 Comment[zh_TW]=貨幣轉換與計算器
-Keywords=converter,euro,money,cash,calculator
-Keywords[bg]=конвертиране, пари, Евро, калкулатор, обменен, курс, converter, euro, money, cash, calculator
-Keywords[bs]=konvertor,euro,kovac,gotovina,kalkulator
-Keywords[ca]=convertidor,euro,diner,efectiu,calculadora
-Keywords[ca@valencia]=convertidor,euro,diner,efectiu,calculadora
-Keywords[cs]=převod měn,euro,peníze,hotovost,kalkulačka
-Keywords[da]=konvertering,euro,penge,kontanter,lommeregner
-Keywords[de]=converter,euro,geld,bar,rechner,währung,cash
-Keywords[el]=μετατροπέας,euro,χρήματα,λεφτά,υπολογιστής
-Keywords[en_GB]=converter,euro,money,cash,calculator
-Keywords[es]=conversor,euro,dinero,moneda,calculadora
-Keywords[et]=teisendaja,euro,raha,valuuta,kalkulaator
-Keywords[fi]=muunnin,euro,raha,käteinen,laskin
-Keywords[fr]=convertisseur, euro, argent, billet, calculatrice
-Keywords[ga]=tiontaire,euro,eoró,airgead,airgead tirim,áireamhán
-Keywords[gl]=convertedor,euro,divisa,cartos,calculadora
-Keywords[hi]=परिवर्तक, यूरो, पैसा, रुपया, नक़दी, केलकुलेटर, गणक
-Keywords[hne]=बदलइया, यूरो, पैसा, रुपया, नकदी, केलकुलेटर, गनक
-Keywords[hu]=konverter,euro,pénz,cash,kalkulátor
-Keywords[it]=convertitore,euro,soldi,contanti,calcolatore
-Keywords[ja]=換算,ユーロ,通貨,お金,現金,キャッシュ,計算機
-Keywords[km]=កម្មវិធី​បម្លែង អឺរ៉ូ រូបិយបណ្ណ លុយ កម្មវិធី​គណនា
-Keywords[ko]=변환기,유로,돈,계산기,euro
-Keywords[lt]=keitiklis,eurai,pinigai,grynieji,skaičiuotuvas
-Keywords[ml]=മാറ്റപ്പട്ടിക, യൂറോ, പണം, കാശ്, ഗണനി
-Keywords[mr]=परिवर्तक, यूरो, पैसा, रुपया, नकद, गणकयंत्र
-Keywords[nb]=konverterer,euro,valuta,kontanter,kalkulator
-Keywords[nds]=Ümreekner,Euro,Geld,Taschenreekner
-Keywords[nl]=converteren,euro,geld,contanten,rekenmachine
-Keywords[nn]=konvertering,euro,pengar,kontantar,kalkulator
-Keywords[pl]=przekształcanie,euro,pieniądze,waluta,kalkulator,konwerter
-Keywords[pt]=conversor,euro,dinheiro,calculadora
-Keywords[pt_BR]=conversor,euro,dinheiro,calculadora
-Keywords[ro]=convertor,euro,bani,calculator,valută
-Keywords[ru]=converter,euro,money,cash,calculator,деньги,калькулятор,евро
-Keywords[sk]=konvertor,euto,peniaze,hotovosť,kalkulačka
-Keywords[sl]=pretvornik,evro,euro,denar,računalo
-Keywords[sr]=converter,euro,money,cash,calculator,мењач,евро,новац,кеш,калкулатор
-Keywords[sr@ijekavian]=converter,euro,money,cash,calculator,мењач,евро,новац,кеш,калкулатор
-Keywords[sr@ijekavianlatin]=converter,euro,money,cash,calculator,menjač,evro,novac,keš,kalkulator
-Keywords[sr@latin]=converter,euro,money,cash,calculator,menjač,evro,novac,keš,kalkulator
-Keywords[sv]=konvertering,euro,valuta,kontanter,räknare
-Keywords[tr]=dönüştürücü,avro,para,nakit,hesaplayıcı
-Keywords[uk]=перетворювач,євро,гроші,готівка,калькулятор
-Keywords[x-test]=xxconverter,euro,money,cash,calculatorxx
-Keywords[zh_CN]=converter,euro,money,cash,calculator,转换器,欧元,钱,现金,计算器
-Keywords[zh_TW]=converter,euro,money,cash,calculator
+Keywords=converter;euro;money;cash;calculator;
+Keywords[bg]=конвертиране; пари; Евро; калкулатор; обменен; курс; converter; euro; money; cash; calculator;
+Keywords[bs]=konvertor;euro;kovac;gotovina;kalkulator;
+Keywords[ca]=convertidor;euro;diner;efectiu;calculadora;
+Keywords[ca@valencia]=convertidor;euro;diner;efectiu;calculadora;
+Keywords[cs]=převod měn;euro;peníze;hotovost;kalkulačka;
+Keywords[da]=konvertering;euro;penge;kontanter;lommeregner;
+Keywords[de]=converter;euro;geld;bar;rechner;währung;cash;
+Keywords[el]=μετατροπέας;euro;χρήματα;λεφτά;υπολογιστής;
+Keywords[en_GB]=converter;euro;money;cash;calculator;
+Keywords[es]=conversor;euro;dinero;moneda;calculadora;
+Keywords[et]=teisendaja;euro;raha;valuuta;kalkulaator;
+Keywords[fi]=muunnin;euro;raha;käteinen;laskin;
+Keywords[fr]=convertisseur; euro; argent; billet; calculatrice;
+Keywords[ga]=tiontaire;euro;eoró;airgead;airgead tirim;áireamhán;
+Keywords[gl]=convertedor;euro;divisa;cartos;calculadora;
+Keywords[hi]=परिवर्तक; यूरो; पैसा; रुपया; नक़दी; केलकुलेटर; गणक;
+Keywords[hne]=बदलइया; यूरो; पैसा; रुपया; नकदी; केलकुलेटर; गनक;
+Keywords[hu]=konverter;euro;pénz;cash;kalkulátor;
+Keywords[it]=convertitore;euro;soldi;contanti;calcolatore;
+Keywords[ja]=換算;ユーロ;通貨;お金;現金;キャッシュ;計算機;
+Keywords[km]=កម្មវិធី​បម្លែង អឺរ៉ូ រូបិយបណ្ណ លុយ កម្មវិធី​គណនា;
+Keywords[ko]=변환기;유로;돈;계산기;euro;
+Keywords[lt]=keitiklis;eurai;pinigai;grynieji;skaičiuotuvas;
+Keywords[ml]=മാറ്റപ്പട്ടിക; യൂറോ; പണം; കാശ്; ഗണനി;
+Keywords[mr]=परिवर्तक; यूरो; पैसा; रुपया; नकद; गणकयंत्र;
+Keywords[nb]=konverterer;euro;valuta;kontanter;kalkulator;
+Keywords[nds]=Ümreekner;Euro;Geld;Taschenreekner;
+Keywords[nl]=converteren;euro;geld;contanten;rekenmachine;
+Keywords[nn]=konvertering;euro;pengar;kontantar;kalkulator;
+Keywords[pl]=przekształcanie;euro;pieniądze;waluta;kalkulator;konwerter;
+Keywords[pt]=conversor;euro;dinheiro;calculadora;
+Keywords[pt_BR]=conversor;euro;dinheiro;calculadora;
+Keywords[ro]=convertor;euro;bani;calculator;valută;
+Keywords[ru]=converter;euro;money;cash;calculator;деньги;калькулятор;евро;
+Keywords[sk]=konvertor;euto;peniaze;hotovosť;kalkulačka;
+Keywords[sl]=pretvornik;evro;euro;denar;računalo;
+Keywords[sr]=converter;euro;money;cash;calculator;мењач;евро;новац;кеш;калкулатор;
+Keywords[sr@ijekavian]=converter;euro;money;cash;calculator;мењач;евро;новац;кеш;калкулатор;
+Keywords[sr@ijekavianlatin]=converter;euro;money;cash;calculator;menjač;evro;novac;keš;kalkulator;
+Keywords[sr@latin]=converter;euro;money;cash;calculator;menjač;evro;novac;keš;kalkulator;
+Keywords[sv]=konvertering;euro;valuta;kontanter;räknare;
+Keywords[tr]=dönüştürücü;avro;para;nakit;hesaplayıcı;
+Keywords[uk]=перетворювач;євро;гроші;готівка;калькулятор;
+Keywords[x-test]=xxconverter;euro;money;cash;calculatorxx;
+Keywords[zh_CN]=converter;euro;money;cash;calculator;转换器;欧元;钱;现金;计算器;
+Keywords[zh_TW]=converter;euro;money;cash;calculator;
